### PR TITLE
#287 Display hotkeys to detach from the container

### PR DESCRIPTION
**Pull request by glendsoza**

### CODE REVIEW

File: pkg/gui/containers_panel.go [hunk 1]
```diff
@@ -409,7 +409,7 @@ func (gui *Gui) handleContainerAttach(g *gocui.Gui, v *gocui.View) error {
 		return gui.createErrorPanel(err.Error())
 	}
 
-	return gui.runSubprocess(c)
+	return gui.runSubprocessWithMessage(c, "By default, to detach from the container press ctrl-p then ctrl-q")
```
Comment (jesseduffield) [inline pkg/gui/containers_panel.go:412]: Can we internationalise this text?

**Review by jesseduffield — CHANGES_REQUESTED** [review]
Nice work :) One thing

**Comment by glendsoza** [comment]
Thanks, @jesseduffield  for taking the time to review, i have internationalised the text as requested 

Please review

**Comment by jesseduffield** [comment]
@glendsoza I've resolved conflicts but I can't push to your branch so I've recreated this in a separate PR: https://github.com/jesseduffield/lazydocker/pull/465

**Comment by jesseduffield** [comment]
PR is great btw! I'll add it in the next release

**Pull request closed without merging**